### PR TITLE
Fix sending messages #1

### PR DESCRIPTION
**Pull request by zhuyuanmao**
This fixes sending messages #1  including unproperly sending messages on the frontend, the messages are not displayed in the correct order and a security issue where user can send messages to any conversation with `conversationId`. 

* The unproperly sending messages is caused by accessing the return value of a Promise object.
* The wrong ordering of messages is caused by the sorting in descending order.
* The security issue is caused by unchecking the participants of the conversation.

### CODE REVIEW

File: client/src/store/utils/thunkCreators.js
Comment (dericf) [inline client/src/store/utils/thunkCreators.js]: Nice! Although this does solve the bug - it doesn't stay consistent with the use of async/await throughout the rest of the app so its preferable to use async/await 🙂

File: server/messenger_backend/views/api/messages.py
Comment (dericf) [inline server/messenger_backend/views/api/messages.py]: Nice! 👍   Just a couple notes here:
- It'll be more readable if you switch the order of the if statement to be a guard clause: e.g. if not conversation then return error. And the message creation logic won't need to be indented.
- returning a 403 is technically more accurate if the user is not part of the conversation.

File: server/messenger_backend/views/api/conversations.py
Comment (dericf) [inline server/messenger_backend/views/api/conversations.py:28]: While this does fix the message ordering - please double check the order of the conversations on the sidebar and make sure they are still in the correct order (they were, and should remain as newest conversations on top)

**Review by dericf — COMMENTED** [review]
Good work on this first PR 👍 

Please address the comments I left inline, and then let us know in the slack channel when you're ready for a second review 🙂

File: server/messenger_backend/views/api/conversations.py
Comment (FilipeGorodscy) [inline server/messenger_backend/views/api/conversations.py:63]: There's another index 0 at line 46, please fix it before merging :)

**Review by FilipeGorodscy — APPROVED** [review]
Good job fixing Deric's comments. I left one comment below. Feel free to merge this PR once you've made the changes :)

**Pull request merged**
